ml-proto/spec/ast.ml

```diff
@@ -20,10 +17,10 @@ and expr' =
   | Loop of expr list
   | Br of var * expr option
   | Br_if of var * expr option * expr
+  | Br_table of var list * var * expr option * expr
```

Comment:
Yeah, putting the default after the `var list` makes monotonic sense.  The only reason to put it before is the vague preference for putting the list at the end, but I think both would work so I'd also be fine putting it after matching the AST here.